### PR TITLE
Refactoring tests

### LINKED ISSUE #27: Write tests
Tests must be written after all other issues will be closed.

### PR DESCRIPTION
**Pull request by viktorprogger**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #27 (just the first step)

### CODE REVIEW

File: config/console.php [hunk 1]
```diff
@@ -1,27 +1,3 @@
...
-//            'password'  => $params['db.password'],
-//            'options'   => [],
-//        ],
-//    ],
-//    \yii\mutex\Mutex::class => Reference::to('mutex'),
-//    'mutex'                 => [
-//        '__class' => \yii\mutex\MysqlMutex::class,
-//    ],
-];
+declare(strict_types=1);
```
Comment (samdark) [inline config/console.php]: That looks suspicious. Why do we need an empty config that doesn't even return empty array?

File: src/Events/BeforePush.php [hunk 2]
```diff
@@ -29,12 +29,12 @@ public function getQueue(): Queue
         return $this->queue;
     }
 
-    public function isPropagationStopped(): bool
+    public function isExecutionnStopped(): bool
```
Comment (samdark) [inline src/Events/BeforePush.php]:
```suggestion
    public function isExecutionStopped(): bool
```

File: config/console.php (hunk 1 above)
Comment (viktorprogger) [reply config/console.php]: Didn't remove it because we will need it for console commands. I'll add empty array here for now.

**Comment by samdark** [comment]
Thank you!

**Pull request merged**